cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -1048,6 +1113,13 @@ public Audio getAudio() {
         return mCameraController.getAudio();
     }
 
+    /**
+     * Gets a boolean which is true if there is at least one overlay view.
+     * @return a boolean which is true if there is at least one overlay view
+     */
+    public boolean hasOverlay() {
```

Comment:
I would remove this, it does not have a super clear meaning. Since we have different overlay targets, and in the future, each view might be allowed to be overlaid only on a specific surface. Let's assume this is always true